src/meta/proto-conv/src/udf_from_to_protobuf_impl.rs

```diff
@@ -129,6 +131,10 @@ impl FromToProto for mt::UserDefinedFunction {
             name: p.name,
             description: p.description,
             definition: udf_def,
+            created_on: match p.created_on {
```

Comment:
I think it's ok. But the old query version does not have this field, this will modify meta data. So @drmingdrmer  Please review this change.

Reply:
`p.created_on` is an `Option`, there is no issue dealing it this way AFAIK. What's your concern? @TCeason

Reply:
> `p.created_on` is an `Option`, there is no issue dealing it this way AFAIK. What's your concern? @TCeason

No problem. I think it's ok. Before this comment I didn't see it before the request review to you. so I cc you to have a look. :)